app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
+  validates :title, presence: true
+  validates :body, presence: true
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
```

Comment:
Since the association is `belongs_to :user`, there is no need to define the `class_name` because it will already be `User` by default. You don't need `foreign_key` either. You can [check here](https://stackoverflow.com/questions/904421/defining-foreign-key-relationships-for-rails-models) for more clarification.